.gitignore

```diff
@@ -22,6 +22,7 @@ habitat/results
 /.direnv
 /.envrc
 results/
+TAGS
```

Comment:
Can you remove this? feel free to keep your local copy but we should not be changing the global `.gitignore` unless inspec is generating something into `TAGS` for everyone.